dali/test/python/test_operator_coco_reader.py

```diff
@@ -25,14 +25,14 @@
 train_annotations = os.path.join(test_data_root, 'db', 'coco', 'instances.json')
 
 test_data = {
-    'car-race-438467_1280.jpg' : 0,
-    'clock-1274699_1280.jpg' : 5,
-    'kite-1159538_1280.jpg' : 6,
-    'cow-234835_1280.jpg' : 17,
-    'home-office-336378_1280.jpg' : 21,
-    'suit-2619784_1280.jpg' : 39,
-    'business-suit-690048_1280.jpg' : 41,
-    'car-604019_1280.jpg' : 59
+    'car-race-438467_1280.jpg' : 17,
+    'clock-1274699_1280.jpg' : 6,
```

Comment:
I would add a test where a custom order with repetition is provided.

Reply:
done